src/main/resources/db/migration/V4__add_sendt_til_gosys_column.sql

```diff
@@ -0,0 +1,2 @@
+alter table oppgave
```

Comment:
Denne setter tilbake_til_gosys til false for alle eksisterende innslag i databasen også, eller..?

Reply:
ja den setter default til false